data/human/campaign events.txt

```diff
@@ -22,7 +22,13 @@ mission "remembrance day"
 		government "Republic"
 		attributes "spaceport"
 	to offer
-		has "event: remembrance day"
+		or
+			and
+				has "event: remembrance day"
+				not "hai start"
+			and
+				day > 19
+				month == 2
```

Comment:
I would suggest that some sort of bespoke experience would be preferable to using the original Remembrance Day text for Hai Start. If we're to assume that the player grew up in Hai space and doesn't have a high amount of knowledge of human space, then most of the context of Remembrance Day may have been lost. You could keep most of the text and just frame it as something that the player is reading or being told when they land during some kind of gathering in honor of Remembrance Day.

Reply:
done but I'm not sure if it's as you envisioned it?
do tell me